### PR TITLE
Created-Issue-template

### PR DESCRIPTION
**Pull request by karlk08**
# Description

This template is designed to standardize the creation of issues across the project, ensuring that all relevant information is provided upfront for better collaboration and problem-solving. Please use this format for all future issues to maintain consistency and clarity.

## Related Issues

- Issue/Request: [#218](https://github.com/pride-land/frontend-pride-land/issues/218)

## Type of change

Please delete options that are not relevant. (All options may be deleted for minor changes)

- [x] This change requires a documentation update

# How Has This Been Tested?

No formal tests have been conducted as this change does not impact functionality or code behavior. This update is primarily related to documentation or a non-functional aspect of the project. 

## Checklist
- [ ] I have ensured the code adheres to the coding standards of the project.
- [x] If changes require documentation updates, I have included them.
- [x] I have performed a self-review of my code.
- [x] New members can easily understand this code, and I have added comments for clarity.
- [ ] The PR is client-ready (all client-related feedback has been addressed).

## Additional Context or Notes
Please review the newly created issue template and provide any feedback or suggestions.
• Ensure that the sections and structure are simple and clear and cover all necessary information.
• To be considered: Are there any fields missing? or do some need further refinement? Is there a better way to organize the template?
• Feel free to suggest changes that may help improve the clarity and usability of the template.
Your input is greatly appreciated! :speech_balloon:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Issue Description
+  [Type of Issue]
+
+## Problem
+  [Describe what the problem we are trying yo solve.]
```
Comment (NabbeunNabi) [inline .github/ISSUE_TEMPLATE]: I think you have a typo here.

Yo should be `to`

Maybe should say

`Describe the problem we are trying to solve`

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- [ ] Add
+- [ ] Requirements
+- [ ] here
+
+## Implementation notes
+ [Suggest how this might be implmented.]
+
+
+### Screenshot (If Applicable)
```
Comment (NabbeunNabi) [inline .github/ISSUE_TEMPLATE]: Is this to show the bug if it is trying to change it or the future feature design? What is this the screenshot of?

**Review by NabbeunNabi — CHANGES_REQUESTED** [review]
Great job @karlk08. Just a couple quick things.

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (karlk08) [reply .github/ISSUE_TEMPLATE]: If applicable, consider including screenshots to provide a clearer visual representation of the issue (example: bug, future features, etc) to enhance understanding and easy code review.

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (karlk08) [reply .github/ISSUE_TEMPLATE]: Thank you for catching this typo.

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (NabbeunNabi) [reply .github/ISSUE_TEMPLATE]: @karlk08 Screenshot or video for context.

#Maybe something like that if applicable

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Problem
+  [Describe the problem we are trying to solve.]
+
+## Success
+  [What should this task accomplish when it's considered completed.]
+
+### Requirements
+
+- [ ] Add
```
Comment (NabbeunNabi) [inline .github/ISSUE_TEMPLATE]: @karlk08 are these spaces correct? I cannot tell? It looks like dash space but I think they need to be together

Comment (karlk08) [reply .github/ISSUE_TEMPLATE]: I removed the space in between dash and square brackets.

**Comment by NabbeunNabi** [comment]
Ship it!

**Pull request merged**